packages/meta/README.md

```diff
@@ -12,6 +12,19 @@ Temporal is a microservice orchestration platform which enables developers to bu
 
 Temporal is a mature technology—it's a fork of Uber's Cadence. Temporal is being developed by [Temporal Technologies](https://temporal.io), a startup by the creators of Cadence.
 
+**Contents**:
```

Comment:
I don't feel like this is needed for such a short file, also GH auto generates TOCs for markdown.